motioneye/sendtelegram.py

```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
         and not snapshot_filename
         or snapshot_filename.startswith('%Y-%m-%d/')
     ):
-        moment = datetime.datetime.strptime(moment, '%Y-%m-%dT%H:%M:%S')
         prefix = moment.strftime('%Y-%m-%d')
         logging.debug('narrowing down still images path lookup to %s' % prefix)
 
-    ## mediafiles.list_media(camera_config, media_type='picture', prefix=prefix, callback=on_media_files)
     fut = utils.cast_future(
         mediafiles.list_media(camera_config, media_type='picture', prefix=prefix)
     )
     fut.add_done_callback(on_media_files)
-
     io_loop.start()
 
 
 def parse_options(parser, args):
+    parser.description='Send Telegram using bot api'
     parser.add_argument('api', help='telegram api key')
     parser.add_argument('chatid', help='telegram chat room id')
-    parser.add_argument('msg_id', help='the identifier of the message')
     parser.add_argument('motion_camera_id', help='the id of the motion camera')
-    parser.add_argument('moment', help='the moment in ISO-8601 format')
+    parser.add_argument('moment', help='the moment in ISO-8601 format', type=datetime.datetime.fromisoformat)
```

Comment:
Line length cap enforced by pre-commit black:
```suggestion
    parser.add_argument(
        'moment',
        help='the moment in ISO-8601 format',
        type=datetime.datetime.fromisoformat,
    )
```